zulipterminal/ui_tools/views.py

```diff
@@ -886,7 +886,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
 
 
 PopUpViewTableContent = Sequence[Tuple[str, Sequence[Union[str,
-                                                           Tuple[str, str]]]]]
+                                                           Tuple[Any, Any]]]]]
```

Comment:
I don't like this change. We should clarify what type this is and define it. I expect it's an urwid type, and possibly one that could be used more widely in ZT.

Reply:
Pinged my concerns :+1: